src/serverless.js

```diff
@@ -145,6 +145,194 @@ class ServerlessComponent extends Component {
     this.state = {}
     console.log(`Removed Tencent ${CONFIGS.compFullname}`)
   }
+
+  async list_alias(inputs) {
+    try {
+      const credentials = this.getCredentials()
+      const region = inputs.region || CONFIGS.region
+
+      const alias_params = {}
+      alias_params.functionName = inputs.function
+      alias_params.functionVersion = inputs.version
+      alias_params.namesapce = inputs.namespace
+
+      console.log(`list alias for function ${inputs.function}...`)
+      const scf = new Scf(credentials, region)
+
+      const scfOutput = await scf.listAlias(alias_params)
+      console.log(`list alias for function ${inputs.function}...`)
+
+      const aliases = scfOutput.Aliases
+      const listAlias = {
+        alias: []
+      }
+
+      for (let i = 0; i < aliases.length; i++) {
+        const mainVersion = aliases[i].FunctionVersion
+        const addWeights = aliases[i].RoutingConfig.AdditionalVersionWeights[0] || { Weight: 0 }
+        const otherVersion = addWeights.Version || null
+        const otherWeight = Number(addWeights.Weight).toFixed(1) || 0
+        const mainWeight = (1 - otherWeight).toFixed(1)
+
+        const alias_unit = {}
+        alias_unit[aliases[i].Name] = {}
+        const weight = {}
+        weight[mainVersion] = mainWeight
+        if (otherVersion) {
+          weight[otherVersion] = otherWeight
+        }
+        alias_unit[aliases[i].Name].weight = JSON.stringify(weight)
+
+        listAlias.alias.push(alias_unit)
+      }
+      console.log(listAlias)
+      return listAlias
+    } catch (e) {
+      return {
+        requestId: e.reqId,
```

Comment:
All errors except api error in components code should throw `TypeError` like:

```
throw new TypeError(
        'xxx',
        `xxx`
 )
```

If you do not throw error, users will get success status even though they get the error message for cli.